src/sentry/static/sentry/app/components/modals/addDashboardWidgetModal.tsx

```diff
@@ -250,7 +308,7 @@ class AddDashboardWidgetModal extends React.Component<Props, State> {
                 disabled={state.loading}
                 busy={state.loading}
               >
-                {t('Add Widget')}
+                {t(isUpdatingWidget ? 'Update Widget' : 'Add Widget')}
```

Comment:
I think for the translation parser to work this needs to be:
```suggestion
          {isUpdatingWidget ? t('Update Widget') : t('Add Widget')}
```